test-docker.sh

```diff
@@ -28,7 +28,7 @@ while ! nc -zvn 127.0.0.1 7777 &>/dev/null; do
     sleep .2
 done
 
-timeout 1800 python3 -u tests/integration/test.py |& tee "logs/${run_id}_test.txt"
+timeout 600 python3 -u tests/integration/test.py |& tee "logs/${run_id}_test.txt"
```

Comment:
Can we move this value to a constant like `TIMEOUT_SECONDS` and use that here?

Reply:
Fixed that by creating a variable `TIMEOUT_SECONDS=600`

Reply:
Ok